crates/autopilot/src/arguments.rs

```diff
@@ -262,6 +267,7 @@ impl std::fmt::Display for Arguments {
             auction_update_interval,
             max_settlement_transaction_wait,
             s3,
+            protocol_fee_exempt_addresses: protocol_fee_discounted_addresses,
```

Comment:
nit: any good reason to have a different name for displaying? otherwise I would keep it 1:1